package.json

```diff
@@ -35,12 +35,9 @@
       "db": {
         "kind": "sql"
       },
-      "uaa": {
-        "kind": "xsuaa"
-      },
       "auth": {
         "[development]": {
-          "strategy": "dummy"
+          "kind": "dummy-auth"
```

Comment:
Isn't that a breaking change for our customers if they now need to specify the kind instead of the strategy?

Reply:
No, we don't want to be breaking, i.e. the old `strategy:dummy` still has to work in the runtime.  Please check if this is the case.

This is about showcasing the _recommended_ approach, which is why we change it.

Reply:
got it

Reply:
> No, we don't want to be breaking, i.e. the old `strategy:dummy` still has to work in the runtime. Please check if this is the case.
> 
> This is about showcasing the _recommended_ approach, which is why we change it.

Still works (with depreciation warning)